javascript/apps/taiga/src/assets/i18n/user_settings/en-US.json

```diff
@@ -20,5 +20,39 @@
   "language": {
     "select": "Select platform language",
     "default": "- Default"
+  },
+  "perefences": {
+    "page_title": "{{ username }}’s preferences settings"
+  },
+  "account": {
+    "title": "Your account",
+    "page_title": "{{ username }}’s account settings",
+    "a11y_title": "{{ username }} user account settings",
+    "subtitle": "Change basic settings about your Taiga account.",
+    "delete": {
+      "msg": "Sorry to see you go, <span class='username'>@{{ username }}</span>.",
+      "explain": {
+        "title": "What will happen:",
+        "access": "You will lose access to Taiga",
+        "delete_data": "We will delete your personal data and your name will disappear from the platform.",
+        "shared": "The content you've created shared with other people will remain in the platform."
+      },
+      "confirm_msg": "I understand that the deletion is final and there will be no way to restore your account.",
+      "delete_account_action": "Delete my account",
+      "confirm_delete_error": "Please, confirm that you understand the consequences of deleting your account.",
+      "confirmation_modal": {
+        "title": "Before deleting your account",
+        "description": "There are some issues that need your attention before deleting your accout",
+        "caution": "Caution!",
+        "keep": "Keep my account",
+        "confirm": "Delete my account",
+        "only_admin_projects": "{projects, plural, one {You are the <strong>only administrator</strong> in <strong>one project</strong>.} other {You are the <strong>only administrator</strong> in <strong>some projects</strong>.}}",
+        "only_admin_workspaces": "{workspaces, plural, one {You are the <strong>only member</strong> in <strong>one workspaces</strong>.} other {You are the <strong>only member</strong> in <strong>some workspaces</strong>.}}",
+        "project_description": "Your role as a project administrator will be moved to another member of the workspace.",
+        "workspace_description": "This workspace will be deleted, along with the projects within them. You can invite somebody to be ws member.",
```

Comment:
When it's more than 1 workspaces it should say 'Those workspaces...'
![image](https://github.com/taigaio/taiga/assets/5970959/f32c269d-8a76-4b28-9cf4-691c25e81a9d)

![image](https://github.com/taigaio/taiga/assets/5970959/790eae06-c37f-42a7-8df2-5b6babdfc65f)